wiki/Options/en.md

```diff
@@ -1,525 +1,162 @@
-| colspan=2|Ignore beatmap hitsounds                                                                                     |
-| colspan=2|When enabled, only your own selected skin's hit-sounds will be played. Beatmap's hit-sounds will be ignored. |
+| Name                     | Effect                                                                                                                         |
+|--------------------------|--------------------------------------------------------------------------------------------------------------------------------|
+| Volume Control           | Master: Controls all aspects<br>Music: Affects only the music<br>Effect: Affects things such as hit sounds and in-game sounds. |
+| Ignore beatmap hitsounds | Favor hitsounds supplied by the current skin, rather than a beatmap's pack-in hitsounds.                                       |
 
 ### Universal Audio Offset
 
 ***[Detailed calibration method.](Universal_Offset "wikilink")***
 
-If, in every single beatmap you play, you hear the sound effects out of sync with the song, you might need to adjust this value. Click and hold on the selector, then drag left or right. You can also place the cursor on the selector and move it using the left and right arrow keys. Continue adjusting until you find the position where the two different sounds are heard simultaneously. Alternatively, you can use Run Offset Wizard for a more graphical representation of ticks.
+If, in every single beatmap you play, you hear the sound effects out of sync with the song, you might need to adjust this value. Click and hold on the selector, then drag left or right. You can also place the cursor on the selector and move it using the left and right arrow keys. Continue adjusting until you find the position where the two different sounds are heard simultaneously. Alternatively, you can use _Run Offset Wizard_ for a more graphical representation of ticks.
 
-Unlike Local Song Offset (specific to that difficulty only) , this offset affects *every* beatmaps you play. If you received an Online Offset, you can \[1\] manually adjust it yourself for the difficulty you are playing using "Plus" (=/+) or "Minus" (-/\_) to increase/decrease the beatmap's offset by 5ms (holding alt while pressing Plus or Minus will allow you to change the offset by 1ms) or \[2\] go to the Universal Audio Offset and adjust it yourself then revert it back to normal upon finishing the said beatmap.
+Unlike Local Song Offset (specific to that difficulty only) , this offset affects *every* beatmap you play. If you received an Online Offset, you can manually adjust it yourself for the difficulty you are playing using "Plus" (=/+) or "Minus" (-/\_) to increase/decrease the beatmap's offset by 5ms (holding alt while pressing Plus or Minus will allow you to change the offset by 1ms).
```

Comment:
replace this `playing using "Plus" (=/+) or "Minus" (-/\_) to increase/decrease` with ``playing using `+` or `-` to increase/decrease``

---

also replace this `(holding alt while pressing Plus or Minus will allow you to change the offset by 1ms).` with ``(holding `Alt` while pressing `+` or `-` will allow you to change the offset by 1ms).``